example/ios/KlaviyoReactNativeSdkExampleExtension/NotificationService.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  NotificationService.swift
+//  KlaviyoReactNativeSdkExampleExtension
+//
+//  Created by Ajay Subramanya on 4/9/24.
+//
+
+import KlaviyoSwiftExtension
+import UIKit
+import UserNotifications
+
+// MARK: notification service extension implementation.
+
+/// When push payload is marked as there being mutable-content this service
+/// (more specifically `didReceiveNotificationRequest` ) is called to perform
+/// tasks such as downloading images and attaching it to the notification before it's displayed to the user.
+///
+/// There is a limited time before which `didReceiveNotificationRequest`  needs to wrap up it's operations
```

Comment:
nit: `There is limited time`, the `a` in this sentence looks unnecessary

Reply:
Same remove those comments.